com.unity.cinemachine/Samples~/Input System/Split Screen Multiplayer/Split Screen Multiplayer.unity

```diff
@@ -0,0 +1,1155 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
```

Comment:
Object naming is confusing.  This is called FreeLook but it isn't one.  It's more like a Lobby camera, I think.  Why not use that naming paradigm?  While you're at it, you can change the name of Center too.

Or maybe put them both under a "Lobby" GameObject, along with the main camera (which is really a Lobby camera).

![image](https://user-images.githubusercontent.com/6424658/231298348-a600b133-2edf-495a-8054-f9025f2be5eb.png)

Reply:
Also, why "plot"?  What does it mean?  As in "plot of land"?  Maybe "Pedestal", or "Start Point"?

Reply:
It bothers me a little that the lobby camera remains active after the first player drops

Reply:
1. Changed the name.
2. Fixed